.github/workflows/scanner-build.yaml

```diff
@@ -59,9 +105,10 @@ jobs:
         push_scanner_image_set ${{ matrix.goarch }}
 
   push-manifests:
-    runs-on: ubuntu-latest
     needs:
+    - define-job-matrix
```

Comment:
You don't need to repeat `define-job-matrix`, as it is already a dependency of the `build-and-push` job.